website/cue/reference/releases/0.37.0.cue

```diff
@@ -31,6 +31,12 @@ releases: "0.37.0": {
 		  for this contribution.
 		"""
 
+	known_issues: [
+		"""
+			The `parse_ddtags` setting added to the datadog_agent incorrectly parses the tags into an object instead of an array.
```

Comment:
```suggestion
			The `parse_ddtags` setting added to `datadog_agent` incorrectly parses the tags into an object instead of an array.
```
Or "the Datadog Agent"

Reply:
Ah, here I intend to reference the `datadog_agent` Vector Source (not the Datadog Agent).

Would it suffice to change that to

```
added to the `datadog_agent` Source
```

?